src/Sulu/Bundle/WebsiteBundle/EventSubscriber/AnalyticsSerializeEventSubscriber.php

```diff
@@ -50,5 +50,11 @@ public function onPostSerialize(ObjectEvent $event)
             $value->domains = true;
             $event->getVisitor()->visitProperty($metadata, $value, $event->getContext());
         }
+
+        $content = $analytics->getContent();
```

Comment:
Why is this necessary here? Tests were not failing before, and the update has already been merged, hasn't it?